map/search_mark.cpp

```diff
@@ -208,9 +231,7 @@ SearchMarkPoint::SearchMarkPoint(m2::PointD const & ptOrg)
 
 m2::PointD SearchMarkPoint::GetPixelOffset() const
 {
-  if (!IsHotel())
-    return {0.0, 4.0};
-  return {0.0, 0.0};
+  return {0.0, 4.0};
```

Comment:
Any ideas why it was made like this?

Reply:
![](https://github.com/user-attachments/assets/2735c0c6-ce10-4927-b7ed-87c6ea852036)

Hotels icons were centered inside the PNG. The rest of the icons had a shadow so they were not centered in the PNG. That's why there was a different offset. 

After this PR all icons have a shadow and are offset the same in the SVG.